androidApp/src/androidMain/kotlin/org/jetbrains/kotlinconf/android/MainActivity.kt

```diff
@@ -28,13 +27,8 @@ class MainActivity : ComponentActivity() {
 
         processIntent(intent)
 
-        val permissionHandler = PermissionHandler(activity = this)
-        val activityModule = module {
-            single<PermissionHandler> { permissionHandler }
-        }
-        KoinPlatformTools.defaultContext()
-            .get()
-            .loadModules(listOf(activityModule))
+        KoinPlatformTools.defaultContext().get()
```

Comment:
`KoinPlatformTools.defaultContext().get()` can be shorter to `KoinPlatform.getKoin()`. It's the same

Reply:
Oops, yes, I even used that elsewhere. Fixed!